### ISSUE TITLE
Missing dependency 'pkg-config'

### ISSUE THREAD

**Issue by tdegeus** [conda-forge/pkgconfig-feedstock#14]
Issue:

On appveyor I'm getting the following error:

```
OSError: pkg-config is not installed
487
```

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
   14_mkl    conda-forge
liblapack                 3.8.0                    14_mkl    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2  
m2w64-pkg-config          0.29.1                        2  
menuinst                  1.4.16           py37he774522_0  
mkl                       2019.4                      245  
msys2-conda-epoch         20160418                      1  
numpy                     1.17.3           py37hc71023c_0    conda-forge
openssl                   1.1.1d               hfa6e2cd_0    conda-forge
pip                       19.3.1                   py37_0  
pkgconfig                 1.3.1                 py37_1001    conda-forge
powershell_shortcut       0.0.1                         2  
pybind11                  2.4.3            py37he980bc4_0    conda-forge
pycosat                   0.6.3            py37hfa6e2cd_0  
pycparser                 2.19                     py37_0  
pyopenssl                 19.1.0                   py37_0  
pysocks                   1.7.1                    py37_0  
python                    3.7.3                h510b542_1    conda-forge
pywin32                   223              py37hfa6e2cd_1  
requests                  2.22.0                   py37_0  
ruamel_yaml               0.15.46          py37hfa6e2cd_0  
setuptools                41.6.0                   py37_0  
six                       1.13.0                   py37_0  
sqlite                    3.30.1               he774522_0  
tqdm                      4.38.0                     py_0  
urllib3                   1.24.2                   py37_0  
vc                        14.1                 h0510ff6_4  
vs2015_runtime            14.16.27012          hf0eaf9b_0  
wheel                     0.33.6                   py37_0  
win_inet_pton             1.1.0                    py37_0  
wincertstore              0.2                      py37_0  
xtensor                   0.20.10              h1ad3211_0    conda-forge
xtl                       0.6.8                h1ad3211_0    conda-forge
yaml                      0.1.7                hc54c509_2 
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : None
       user config file : C:\Users\appveyor\.condarc
 populated config files : C:\Users\appveyor\.condarc
          conda version : 4.7.12
    conda-build version : not installed
         python version : 3.7.3.final.0
       virtual packages : 
       base environment : C:\myname-conda  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\myname-conda\pkgs
                          C:\Users\appveyor\.conda\pkgs
                          C:\Users\appveyor\AppData\Local\conda\conda\pkgs
       envs directories : C:\myname-conda\envs
                          C:\Users\appveyor\.conda\envs
                          C:\Users\appveyor\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.7.12 requests/2.22.0 CPython/3.7.3 Windows/10 Windows/10.0.14393
          administrator : True
             netrc file : None
           offline mode : False
```
</details>

**Comment by tdegeus** [comment]
The workaround that works for me is 

```
conda install pkg-config -c conda-forge
```

But I think it warrant a fix here.

**Comment by ocefpaf** [comment]
It was removed in https://github.com/conda-forge/pkgconfig-feedstock/pull/7/commits/2488fd099939c300b4d6991214815008bec1a059 but it looks like an accident.

Do you want to send a PR to fix that?

**Comment by tdegeus** [comment]
Done! #15 
@ocefpaf

**Comment by tdegeus** [comment]
Note that my issue is at *runtime*.

**Comment by ocefpaf** [comment]
Fixed in #15.
